lib/updatehub/updatehub.c

```diff
@@ -687,6 +687,12 @@ enum updatehub_response updatehub_probe(void)
 			goto cleanup;
 		}
 
+		if (metadata_any_boards.objects[1].objects.sha256sum >
```

Comment:
are you missing a `strlen()` or `sizeof()` here?

Reply:
I was just wondering the same

Reply:
I think you are missing an `strlen()`, since the object is:

```
struct resp_probe_objects {
        const char *mode;
        const char *sha256sum;
        int size;
};
```